app/src/main/res/values-it/strings.xml

```diff
@@ -357,6 +357,6 @@
     <string name="status_wallet_connect_wrongversion">Versione del nodo incompatibile - si prega di aggiornare!</string>
 
     <string name="menu_info">Dettagli</string><!--Changed to: Show Secrets!-->
```

Comment:
could please (re-)translate this: it was changed from "Details" to "Show Secrets!"